recipes/jupyter-docx-bundler/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "jupyter-docx-bundler" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9457377f8d25dcde1a4de241f31ddc31d18d503f00ccf02823393729196584a1
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

If (and only if) it is made `noarch`:
```suggestion
    - python >=3.6
```